code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -272,28 +272,44 @@ var/global/chicken_count = 0
 	else
 		SSobj.processing.Remove(src)
 
-/obj/udder
 
-/obj/udder/New()
+/obj/item/udder

```

Comment:
@Incoming5643 yeah it was literally the only /obj in the code.

@Core0verload That's not really how udders work...
I mean it wouldn't be hard at all to move the udder code up in order to allow udder transplants so you can milk your crewm-- nope nope nope no forget i said anything